src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -873,6 +873,27 @@ public JSONObject updateTransitionExecution(
         return new JSONObject(jsonResponse);
     }
 
+    /**
+     * Send heartbeat for a manual execution,
```

Comment:
"Send heartbeat for a manual execution to signal that we are still working on it. Must be done at minimum once every 60 seconds or the transition execution will time out"